src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  health = 100;
+  hidden = false;
```

Comment:
only Herbivore has to have this property. it would be better to add it only in that class